charm/kafka/reactive/storage.py

```diff
@@ -9,6 +9,17 @@
 from charmhelpers.core.hookenv import log
 
 
+def init_brokerid(log_dir):
+    broker_path = os.path.join(log_dir, '.broker_id')
+    broker_id = hookenv.local_unit().split('/', 1)[1]
+
+    if not os.path.exists(broker_path):
+        os.makedirs(log_dir)
```

Comment:
`os.makedirs(log_dir, exist_ok=True)` or this will raise an exception if the log dir already exists.